packages/twenty-server/src/engine/core-modules/google-map/google-api-map.module.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpModule } from '@nestjs/axios';
+import { Module } from '@nestjs/common';
+
+import { TokenModule } from 'src/engine/core-modules/auth/token/token.module';
+import { GoogleApiMapController } from 'src/engine/core-modules/google-map/controller/google-api-map.controller';
+import { GoogleApiMapService } from 'src/engine/core-modules/google-map/services/google-api-map.service';
+import { WorkspaceCacheStorageModule } from 'src/engine/workspace-cache-storage/workspace-cache-storage.module';
+
+@Module({
+  imports: [HttpModule, WorkspaceCacheStorageModule, TokenModule],
+  controllers: [GoogleApiMapController],
+  providers: [GoogleApiMapService],
+  exports: [],
+})
+export class GoogleApiMapModule {}
```

Comment:
This looks great, I'll start the review, but as a first comment: we try to have an agnostic approach to prepare for the future when we can (lookup `driver` in the code base to see the driver-based approach we have in different places), so even if for the next year we end up implementing just Google, it's good to remain open to the future. In practise, you don't have to implement a driver-based system for now as that would be premature optimization, but you could name the different files in a more generic way (GeoMapModule? I feel like "Map" is not clear enough even though it must be in the name)